### PR TITLE
fix(collapse): update focus style to align with halo design

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description
Focus ring of collapse didn't align to halo design that should be in #334BFF color.

Fixes # (issue)
DME-2091

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by kosol-komenek-rft** [comment]
look good for me. 👍

**Review by kosol-komenek-rft — DISMISSED** [review]
look good for me. 👍

File: packages/elemental-theme/src/custom-elements/ef-collapse.less [hunk 1]
```diff
@@ -24,6 +24,8 @@
     padding: 0;
     &[focused='visible'] {
       background-color: @button-hover-background-color;
+      outline: 1px solid @button-hover-border-color;
```
Comment (Theeraphat-Sorasetsakul) [inline packages/elemental-theme/src/custom-elements/ef-collapse.less]: The focus color should be "334bff" like other elements such as checkbox.

**Review by Theeraphat-Sorasetsakul — APPROVED** [review]
LGTM.

**Pull request merged**